config/locales/en.json

```
@@ -502,7 +491,7 @@
   "dashboard.consent": "has given consent for Service Canada to obtain personal information about them.",
```

```suggestion
  "dashboard.consent": "has consented to you sending their personal information to Service Canada.",
```
Changed to reflect new consent type.